core/src/test/java/org/axonframework/commandhandling/SpawningNewAggregateTest.java

```diff
@@ -65,7 +65,7 @@
     @SuppressWarnings("unchecked")
     @Before
     public void setUp() throws Exception {
-        commandBus = new SimpleCommandBus();
+        commandBus = SimpleCommandBus.builder().build();
```

Comment:
Does it make sense to have something like: `SimpleCommandBus.default()` which would invoke `builder().build()`. It's just a minor thing.

Reply:
Could definitely be worthwhile. In interest of time, I wont do this right now though.
Similarly as with the previous point, let's leave this as a future exercise.